osu.Game/Online/Chat/DrawableLinkCompiler.cs

```diff
@@ -52,6 +62,32 @@ private void load(OsuColour colours)
 
         protected override IEnumerable<Drawable> EffectTargets => Parts;
 
+        public MenuItem[] ContextMenuItems
+        {
+            get
+            {
+                List<MenuItem> items = new List<MenuItem>();
+                bool hasTooltip = TooltipText.ToString() != "";
+
+                string text = hasTooltip ? TooltipText.ToString() : GetUrlFromPart(Parts);
+
+                bool isChannelorLobbyLink = text.Contains("osu://chan") || text.Contains("osump://");
+
+                if (!isChannelorLobbyLink)
+                {
+                    items.Add(new OsuMenuItem("Open", MenuItemType.Highlighted, () => host.OpenUrlExternally(text)));
```

Comment:
I think the expected UX for highlighted menu items is that it behaves like left click. In this case, it should open the in-game overlays.